setup.py

```diff
@@ -57,5 +51,8 @@
     platforms=["any"],
     test_suite='faker.tests',
     zip_safe=zip_safe,
-    install_requires=install_requires
+    extras_requires={
```

Comment:
classical typo, the tailing 's' is too much

see `setup.py --version` for the error message

for further convenience and better windows script support, you might want to check out the console_scripts entry-point instead of using a normal script